Microsoft.Toolkit.Uwp.Connectivity/Network/ConnectionInformation.cs

```diff
@@ -30,22 +30,20 @@ public class ConnectionInformation
         /// Updates  the current object based on profile passed.
         /// </summary>
         /// <param name="profile">instance of <see cref="ConnectionProfile"/></param>
-        public virtual void UpdateConnectionInformation(ConnectionProfile profile)
+        public void UpdateConnectionInformation(ConnectionProfile profile)
```

Comment:
Would this be considered a breaking change?

Reply:
I consider that method as not being sufficient for anyone to override.

How and why it was marked virtual is a mystery to me.

For anyone insane enough to inherit from connection information object yes it's a bragging change though I'm at loss as to why one would.

Reply:
So it sound like this is fixing a bug then instead ;)

Approved